docs/Learning-Environment-Create-New.md

```diff
@@ -343,11 +343,12 @@ with our Agent code.
 
 1. Select the **RollerAgent** GameObject to show its properties in the Inspector
     window.
-2. Change **Decision Interval** from `1` to `10`.
-3. Drag the Target GameObject from the Hierarchy window to the RollerAgent
+2. Add the Decision Requester Parameters script with the Add Component button from the RollerAgent Inspector.
+3. Change **Decision Period** from `1` to `20`.
```

Comment:
```suggestion
3. Change **Decision Period** from `1` to `10`.
```
The old docs said 10. Maybe 20 works too though?

Reply:
![Screenshot (19)](https://user-images.githubusercontent.com/39962268/73913462-1ecdd680-48dd-11ea-98ad-57a1d5d46fdc.png)

It says Decision period can be Max 20 . Maybe 10 was earlier. And I tried to change it to 50 but it again changed back to 20.

Reply:
OK, I think I understand the confusion; it's not describing the range, it's describing the value you should set. I think this is a better wording:
```suggestion
3. Change **Decision Period** to `20`.
```